bundle/artifacts/build.go

```diff
@@ -33,9 +33,23 @@ func (m *build) Apply(ctx context.Context, b *bundle.Bundle) error {
 		return fmt.Errorf("artifact doesn't exist: %s", m.name)
 	}
 
-	if artifact.Notebook != nil {
-		return bundle.Apply(ctx, b, notebook.Build(m.name))
+	if artifact.File == "" && artifact.BuildCommand == "" {
+		return fmt.Errorf("artifact %s misconfigured: 'file' or 'build' property is requried", m.name)
 	}
 
-	return nil
+	// If artifact file is explicitly defined, skip building the artifact
+	if artifact.File != "" {
+		return nil
+	}
+
+	// If artifact path is not provided, use current dir
+	if artifact.Path == "" {
+		path, err := os.Getwd()
```

Comment:
We should use `b.Config.Root`. It is possible to run bundle commands outside of the bundle tree by setting `DATABRICKS_BUNDLE_ROOT` and then this wouldn't work.